tests/safety/test_subaru.py

```diff
@@ -7,35 +7,46 @@
 from functools import partial
 
 
-MSG_SUBARU_Brake_Status     = 0x13c
-MSG_SUBARU_CruiseControl    = 0x240
-MSG_SUBARU_Throttle         = 0x40
-MSG_SUBARU_Steering_Torque  = 0x119
-MSG_SUBARU_Wheel_Speeds     = 0x13a
-MSG_SUBARU_ES_LKAS          = 0x122
-MSG_SUBARU_ES_LKAS_ANGLE    = 0x124
-MSG_SUBARU_ES_Brake         = 0x220
-MSG_SUBARU_ES_Distance      = 0x221
-MSG_SUBARU_ES_Status        = 0x222
-MSG_SUBARU_ES_DashStatus    = 0x321
-MSG_SUBARU_ES_LKAS_State    = 0x322
-MSG_SUBARU_ES_Infotainment  = 0x323
+MSG_SUBARU_Brake_Status      = 0x13c
+MSG_SUBARU_CruiseControl     = 0x240
+MSG_SUBARU_Throttle          = 0x40
+MSG_SUBARU_Steering_Torque   = 0x119
+MSG_SUBARU_Wheel_Speeds      = 0x13a
+MSG_SUBARU_ES_LKAS           = 0x122
+MSG_SUBARU_ES_LKAS_ANGLE     = 0x124
+MSG_SUBARU_ES_Brake          = 0x220
+MSG_SUBARU_ES_Distance       = 0x221
+MSG_SUBARU_ES_Status         = 0x222
+MSG_SUBARU_ES_DashStatus     = 0x321
+MSG_SUBARU_ES_LKAS_State     = 0x322
+MSG_SUBARU_ES_Infotainment   = 0x323
+MSG_SUBARU_ES_UDS_Request    = 0x787
+MSG_SUBARU_ES_HighBeamAssist = 0x121
+MSG_SUBARU_ES_STATIC_1       = 0x22a
+MSG_SUBARU_ES_STATIC_2       = 0x325
+
 
 SUBARU_MAIN_BUS = 0
 SUBARU_ALT_BUS  = 1
 SUBARU_CAM_BUS  = 2
 
 
 def lkas_tx_msgs(alt_bus, lkas_msg=MSG_SUBARU_ES_LKAS):
-  return [[lkas_msg,                    SUBARU_MAIN_BUS],
-          [MSG_SUBARU_ES_Distance,      alt_bus],
-          [MSG_SUBARU_ES_DashStatus,    SUBARU_MAIN_BUS],
-          [MSG_SUBARU_ES_LKAS_State,    SUBARU_MAIN_BUS],
-          [MSG_SUBARU_ES_Infotainment,  SUBARU_MAIN_BUS]]
-
-def long_tx_msgs():
-  return [[MSG_SUBARU_ES_Brake,         SUBARU_MAIN_BUS],
-          [MSG_SUBARU_ES_Status,        SUBARU_MAIN_BUS]]
+  return [[lkas_msg,                     SUBARU_MAIN_BUS],
+          [MSG_SUBARU_ES_Distance,       alt_bus],
+          [MSG_SUBARU_ES_DashStatus,     SUBARU_MAIN_BUS],
+          [MSG_SUBARU_ES_LKAS_State,     SUBARU_MAIN_BUS],
+          [MSG_SUBARU_ES_Infotainment,   SUBARU_MAIN_BUS]]
+
+def long_tx_msgs(alt_bus):
+  return [[MSG_SUBARU_ES_Brake,          alt_bus],
+          [MSG_SUBARU_ES_Status,         alt_bus]]
+
+def additional_tx_msgs(alt_bus):
```

Comment:
can you call it the same thing as in panda safety (gen2) if it is? same comment about `alt_bus` argument too

Reply:
done